src/tests/multihost/ipa/test_misc.py

```diff
@@ -610,10 +610,16 @@ def test_ssh_expiration_warning(multihost, reset_password, hbac_sshd_rule,
         client.clear_sssd_cache()
 
         # Run ssh
-        cmd = 'su - foobar1@testrealm.test -c " ssh -v ' \
+        # This one does not work RHEL 8
+        # cmd = 'su - foobar1@testrealm.test -c " ssh -v ' \
+        #       '-o StrictHostKeychecking=no -o UserKnownHostsFile=/dev/null ' \
+        #       '-o GSSAPIAuthentication=no -o PasswordAuthentication=no ' \
+        #       '-l foobar1@testrealm.test localhost \'whoami\' " 2>&1'
+
+        cmd = 'sudo -u foobar1@testrealm.test ssh -v ' \
```

Comment:
Simply adding this line do the magic :

cmd = "sudo -u foobar1 ssh -o StrictHostKeychecking=no 127.0.0.1 <<< whoami"


It is working for both 8.9 and 9.2

Reply:
https://ci-jenkins-csb-idmops.apps.ocp-c1.prod.psi.redhat.com/job/User-Tools/job/trigger-test-suite-tool/2475/console    for 8.9 

https://ci-jenkins-csb-idmops.apps.ocp-c1.prod.psi.redhat.com/job/User-Tools/job/trigger-test-suite-tool/2476/console    for 9.3

Reply:
It might be, I just do not want to risk that: if the key for some reason does not work the test will get stuck with password prompt hanging there until the whole suite dies on timeout.

This works on 8.9
2023-05-24T06:18:40 PASSED src/tests/multihost/ipa/test_misc.py::Testipabz::test_ssh_expiration_warning
2023-05-24T06:18:40 PASSED src/tests/multihost/ipa/test_misc.py::Testipabz::test_ssh_expired_warning